config-sample.ini

```diff
@@ -21,3 +21,9 @@ https=false
 # path to certificate (run "bash bin/generate-cert.sh" to generate self-signed certificate). Relevant only if https=true
 certPath=
 keyPath=
+# setting to give trust to reverse proxies, a comma-separated list of trusted rev. proxy IPs can be specified (CIDR notation is permitted),
+# alternatively 'true' will make use of the leftmost IP in X-Forwarded-For, ultimately an integer can be used to tell about the number of hops between
+# Trilium (which is hop 0) and the first trusted rev. proxy. 
+# once set, expressjs will use the X-Forwarded-For header set by the rev. proxy to determinate the real IPs of clients.
+# expressjs shortcuts are supported: loopback(127.0.0.1/8, ::1/128), linklocal(169.254.0.0/16, fe80::/10), uniquelocal(10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16, fc00::/7)
+trustedReverseProxy=false
```

Comment:
A default value of "false" will cause the `app.set()` to be called, I think it would be better if the default setting would do nothing instead. Either set it to empty string or catch "false" value in condition?

Reply:
> catch "false" value in condition?

I am testing whether it's false or the key doesn't exist, if it's the case app.set() won't be called, line 48 in src/www
On the other hand I'm testing twice whether the key is true (line 48-49) because otherwise it would try to trim a boolean type, I'm not sure how to avoid this.
Otherwise I think I could compact it into one single 'if' statement if preferred (I'm not sure I do).
I'm not familiar with JavaScript tbh.